src/modules/triggers/exports.bicep

```diff
@@ -0,0 +1,57 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Required. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('Required. The ID of the storage account.')
+param storageAccountId string
+
+var PipelineName = 'extract_parquet'
```

Comment:
Aren't all the pipelines named after the container they monitor (e.g., `ms-cm-exports_extract`)?

Reply:
Also, ADF doesn't like dashes in names.